File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F02.P2.java
class Main {
    static byte probe() {
        // BEGIN PROBE F02.P2
        byte[] buf = new byte[16];
        buf[0] = 1;
        return buf[0];
        // END PROBE F02.P2
    }

    public static void main(String[] args) {
        System.out.println(probe());
    }
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/cpp/F02.P2.cpp
#include <cstdint>
#include <cstdio>

std::uint8_t probe() {
// BEGIN PROBE F02.P2
std::uint8_t buf[16];
buf[0] = 1;
return buf[0];
// END PROBE F02.P2
}

int main() {
    std::printf("%u\n", probe());
}
